MantidQt/CustomInterfaces/test/MuonAnalysisFitDataPresenterTest.h

```diff
@@ -148,6 +148,7 @@ class MuonAnalysisFitDataPresenterTest : public CxxTest::TestSuite {
   }
 
   void test_setAssignedFirstRun_singleWorkspace() {
+    setupGroupPeriodSelections();
```

Comment:
Oops! I misread the code and thought an early call of `setupGroupPeriodSelections` in `test_setSelectedWorkspace` was the name of that latter function, having expected you to have added code for `setupGroupPeriodSelections`.